src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
> Is it really worth avoiding a simple patch which appends the prefix as proposed in this PR for rospkg as well as rospack?

The difference is that `rospkg` is installed out side of `/opt` and conflicts with the one in Ubuntu. You install one or the other, not both. I mentioned a specific case where this was a problem above. This is one of the only, if not the only, patch that I think makes a big difference in terms of usability when both are installed.

> The other option which allows things to work by default but won't interfere with the normal flow is that the environment variables only have a default value if unset.

You mean this? https://github.com/ros-infrastructure/rospkg/pull/102#discussion_r74334509

I don't think that's a good user experience. Consider this common practice of ROS users:

```
$ rospack list
... lots of things
$ export ROS_PACKAGE_PATH=$ROS_PACKAGE_PATH:`pwd`
$ rospack list
... only the packages under the current directory (not the packages in `/usr`)
```

(assume I'm invoking the same interface in `rospkg` rather than using `rospack list`)